core-bundle/src/Resources/contao/forms/FormFileUpload.php

```diff
@@ -179,23 +181,23 @@ public function validate()
 
 		if ($arrImageSize = @getimagesize($file['tmp_name']))
 		{
-			$intImageWidth = Config::get('imageWidth');
+			$intImageWidth = $this->getMaximumImageWidth();
```

Comment:
```suggestion
			$intImageWidth = $this->maxImageWidth ?: Config::get('imageWidth');
```

imho we do not need a separate method for that